src/test/common/ios/xcodeproj.test.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {Xcodeproj} from "../../../common/ios/xcodeproj";
+
+import * as assert from "assert";
+import * as path from "path";
+import * as Q from "q";
+
+suite("xcodeproj", function() {
+    suite("commonContext", function() {
+        test("should look in the correct location for xcodeproj files and return one", function() {
+            const projectRoot = "projectRoot";

```

Comment:
I think we should use a path that's a possible real path here like /Users/my_username/my_project/